.github/ISSUE_TEMPLATE/roadmap-item.md

```diff
@@ -0,0 +1,31 @@
+---
+name: 'Roadmap Item'
+about: Create roadmap item for this project
+title: Roadmap Item =description=
+labels: kind/roadmap
```

Comment:
should also include triage/needs-triage but that label does not exist (cf. https://github.com/o3de/sig-release/issues/79#issuecomment-1379023125)

Reply:
Agree to add the triage/needs-triage as part of the issue template. I'll update the PR this week. 

About the label doesn't exist, my response can be read here: https://github.com/o3de/sig-release/issues/79#issuecomment-1381371262